dashboard/lib/metrics/events.rb

```diff
@@ -46,11 +46,21 @@ def log_event(user: nil, event_name:, event_value: nil, metadata: {}, get_enable
       )
       end
 
+      def log_event_with_session(session:, event_name:, event_value: nil, metadata: {})
+        statsig_user = StatsigUser.new({'userID' => session[:statsig_stable_id]})
```

Comment:
Or, what if we just move this line down to 76, where we initialize an empty string for a user. Then if the user is nil, we just init a statsig user with the :statsig_stable_id